pkgs/by-name/ne/netatalk/package.nix

```diff
@@ -47,26 +51,31 @@ stdenv.mkDerivation (finalAttrs: {
     libiconv
     openssl
     pam
+    cracklib
+    cups
+    libtirpc
+    openldap
+    glib
+    dbus
+    cracklib
   ];
 
-  configureFlags = [
-    "--with-bdb=${db.dev}"
-    "--with-ssl-dir=${openssl.dev}"
-    "--with-lockfile=/run/lock/netatalk"
-    "--localstatedir=/var/lib"
+  mesonFlags = [
+    "-Dwith-appletalk=true"
+    "-Dwith-bdb-path=${db.out}"
+    "-Dwith-bdb-include-path=${db.dev}/include"
+    "-Dwith-install-hooks=false"
+    "-Dwith-init-hooks=false"
+    "-Dwith-lockfile-path=/run/lock/"
+    "-Dwith-cracklib=true"
+    "-Dwith-cracklib-path=${cracklib.out}"
   ];
 
-  postInstall = ''
-    sed -i -e "s%/usr/bin/env python%${python3}/bin/python3%" $out/bin/afpstats
-    buildPythonPath ${python3.pkgs.dbus-python}
-    patchPythonScript $out/bin/afpstats
-  '';
-
-  enableParallelBuilding = true;
+  enableParallelBuilding = false;
```

Comment:
Please explain why this is set to `false`. Usually meson builds can be parallelized. If not it probably indicates a bug in one of the meson files.

Reply:
Likely does and I did report it to the project maintainer (opened an issue to track now as well). I can flip it back once that bug is resolved. 

https://github.com/Netatalk/netatalk/issues/1919

Reply:
Thank you. Would be good to add a link to that issue in a comment next to `enableParallelBuilding = false` :)

Reply:
Done

Reply:
Oddly the problem doesnt happen locally for me anymore and CI passed, must have been something odd in my env. Put it back to parallel.